hatchways/team-naan-bread
# IN: Pet Meetup Events

### What this PR does (required):
- merged both https://github.com/hatchways/team-naan-bread/pull/90 and https://github.com/hatchways/team-naan-bread/pull/91 and added updates to integrate them together
- updated the frontend with components and APICalls to edit, delete or attend an event
- added a route in the backend to fetch one event with populated attendees and host.
- updated the event model in both the frontend and backend to use MongoDB's [GeoJSON](https://mongoosejs.com/docs/geojson.html)
### Screenshots / Videos (required):
https://user-images.githubusercontent.com/29197126/135327617-5d8f722c-8b9f-4e0b-ba05-fc6b12fa7aba.mov

### Any information needed to test this feature (required):
- log in as a new user
-  click on events item in the navbar to add a new event or see events nearby.
- if you allow geolocation in the `/events` page, you should fetch all events with in 100km from your location
